edit-post/api/more-menu-item.js

```diff
@@ -0,0 +1,91 @@
+/* eslint no-console: [ 'error', { allow: [ 'error' ] } ] */
+
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import { applyFilters } from '@wordpress/hooks';
+import { isString } from 'util';
```

Comment:
Here's the code that's pulled in with this import statement:

https://github.com/defunctzombie/node-util/blob/master/util.js

...for what can be expressed as `typeof foo === 'string';`

At the very least, we should use the one from Lodash, which is already used elsewhere and is prepared for tree-shaking:

https://lodash.com/docs/4.17.5#isString

Reply:
Not sure where this came from, should've been lodash :)